ts/src/kraken.ts

```diff
@@ -325,6 +325,91 @@ export default class kraken extends Exchange {
                     'ZEC': 'Zcash (Transparent)',
                     'ZRX': '0x (ZRX)',
                 },
+                'withdrawMethods': {
```

Comment:
@pcriadoperez two questions, first why aren't these inside `networks` but the other way around (UNIFIED:ID)? By doing that `networksById` is computed automatically.


Second, why not use `networkIdToCode/networkCodeToID` to parse these values?

Reply:
> found some differences between the naming of the same network depending if they are deposits or withdrawals

@pcriadoperez damn that's bad 😅  I don't get why they would do things like that